introduction/issue-tracking.rst

```diff
@@ -47,13 +47,15 @@ Great! Thank you for taking the time and effort to help improve Qubes! To ensure
 
 6. Do not delete the provided issue template. Fill out every applicable section.
 
-7. Make sure to mention any relevant documentation and other issues you’ve already seen. We don’t know what you’ve seen unless you tell us. If you don’t list it, we’ll assume you haven’t seen it.
+7. Please note that AIs often `hallucinate <https://en.wikipedia.org/wiki/Hallucination_(artificial_intelligence)>`__ about Qubes OS. If you're using an AI to assist you, please check its conclusions against the `official documentation <https://doc.qubes-os.org/>`__.
```

Comment:
the referal to the hallucination's wikipedia page does use the description AI, but still they are just Large Language Models. It is in my opinion of importance to use this terminology to make a clear distinction between intelligence and a sophisticated tooling based on probabilistic systems. I do not have a too strong opinion on this specific use of wording regarding the wikipedia page however :) It is just a suggestion.

Would it be ok if i rebase your PR?